std/node/_fs/_fs_lstat.ts

```diff
@@ -36,9 +36,10 @@ export function lstat(
 
   if (!callback) throw new Error("No callback function supplied");
 
-  Deno.lstat(path)
-    .then((stat) => callback(undefined, CFISBIS(stat, options.bigint)))
-    .catch((err) => callback(err, err));
+  Deno.lstat(path).then(
+    (stat) => callback(undefined, CFISBIS(stat, options.bigint)),
```

Comment:
```suggestion
    (stat) => callback(null, CFISBIS(stat, options.bigint)),
```
(node always passes null if there is no error)